mok.c

```diff
@@ -741,12 +741,12 @@ mirror_one_mok_variable(struct mok_state_variable *v,
 			/*
 			 * Measure this into PCR 7 in the Microsoft format
 			 */
-			efi_status = tpm_measure_variable(v->name, *v->guid,
+			EFI_STATUS tpm_status;
+			tpm_status = tpm_measure_variable(v->name, *v->guid,
 							  FullDataSize, FullData);
-			if (EFI_ERROR(efi_status)) {
-				dprint(L"tpm_measure_variable(\"%s\",%lu,0x%llx)->%r\n",
-				       v->name, FullDataSize, FullData, efi_status);
-				return efi_status;
+			if (EFI_ERROR(tpm_status)) {
+				dprint(L"tpm_measure_variable(\"%s\",%lu,0x%llx)->%r (ignored)\n",
+				       v->name, FullDataSize, FullData, tpm_status);
```

Comment:
I know the codebase isn't great about this and the problem predates your change, but it would be nice to keep lines <80 characters when it's easy to do so.

Reply:
In general I agree, but I also really don't like breaking output strings in the middle, as that makes them harder to grep for.

Reply:
Agreed on the output string, but I'm actually talking about the next line (that supplies the other args to dprint()).

Reply:
Oh, I see.  github's elision of most of the whitespace has fooled me.